File: CSW1/Assignment_6/Q5.java

/*
 * Name:Anish More
 * Registration Number: 2241013365
 * Name of program: Write a java program to copmute x × y without arithmetic operators
 */
import java.util.Scanner;

public class Q5 {

	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		System.out.print("Enter x:");
		int x = sc.nextInt();
		System.out.print("Enter y:");
		int y = sc.nextInt();
		int result = 0;
		int b1;
		for (int i = 1; i <= x; i++) {
			b1 = y;
			while (b1 != 0) {
				int carry = (result & b1) << 1;
				result = result ^ b1;
				b1 = carry;
			}
		}
		System.out.println(x + "x" + y + "=" + result);
	}
}
/*
 * Output--
 * Enter x:5
 * Enter y:6
 * 5x6=30
 * 
 * 
 * Enter x:10
 * Enter y:24
 * 10x24=240
 */

File: PPWC/Minor_Assignment_5/Q5.c
#include <stdio.h>
int main(){
    int a[][3]={4,5,6,7,8,9,1,2,3};
    printf("%d,", *a[2]);
    printf("%d,", a[2][0]);
    printf("%d ", **(a+1+('b'-'a')));
    return 0;
}
/*
Output--
1,1,1
*/
